packages/vite/package.json

```diff
@@ -45,6 +45,7 @@
   "//": "READ .github/contributing.md to understand what to put under deps vs. devDeps!",
   "dependencies": {
     "esbuild": "^0.9.3",
+    "portfinder": "^1.0.28",
```

Comment:
**issue (blocking):** I'm very sorry, but I need to reject your PR in the current state 🙁 

Please read the [Notes on Dependencies](https://github.com/vitejs/vite/blob/main/.github/contributing.md#notes-on-dependencies) section again

So this adds a direct dependency to Vite to only solve a minor port mismatch. We need to find a better solution here.
Even if using an optional lazy loaded dependency I would question this.

Reply:
I've try to fix it without add `portfinder` package as a dependency, but It still cannot be fixed even the implementation is as same as `portfinder`'s code.

You can close or push another commits into this PR to fix it 😢 .

Thanks.

Reply:
I will try, but need to find time for it, maybe today, maybe over weekend 🤷